reana_workflow_controller/config.py

```diff
@@ -50,3 +50,10 @@
 WORKFLOW_QUEUES = {'cwl': 'cwl-default-queue',
                    'yadage': 'yadage-default-queue',
                    'serial': 'serial-default-queue'}
+
+WORKFLOW_ENGINE_SHARED_VOLUME = 'vm'
```

Comment:
Hmm, `vm` may be perhaps confusion-prone, what about using `local`? (or perhaps `host`?)

Reply:
Thanks! I think host is the most representative, I was not sure what to use here 😕